cards/src/main/resources/cards/custom/group2/set_633/basic/spell_steadfast_defense.json

```diff
@@ -64,7 +64,7 @@
       }
     ]
   },
-  "collectible": true,
+  "collectible": false,
```

Comment:
this is a token now right?

Reply:
this isn't a token, but it was a card that Grand explicitly asked to be made uncollectible for now.